cfme/fixtures/v2v.py

```diff
@@ -41,6 +37,292 @@ def conversion_tags(appliance, host_creds):
     tag2 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Method').collections.tags.instantiate(
         display_name='VDDK')
+    host_creds.remove_tags(host_creds.get_tags())
     host_creds.add_tags(tags=(tag1, tag2))
     yield
     host_creds.remove_tags(tags=(tag1, tag2))
+
+
+def get_vm(request, appliance, second_provider, template, datastore='nfs'):
+    """Fixture to provide vm object"""
+    source_datastores_list = second_provider.data.get('datastores', [False])
+    if not source_datastores_list:
+        pytest.skip("No data for source datastores in provider.")
+
+    source_datastore = [d.name for d in source_datastores_list if d.type == datastore][0]
+    collection = second_provider.appliance.provider_based_collection(second_provider)
+    vm_obj = collection.instantiate(random_vm_name('v2v'),
+                                    second_provider,
+                                    template_name=template(second_provider)['name'])
+
+    request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
```

Comment:
There's no need for a lambda here, just pass the function object without calling it.